scripts/generate-docs-admin.ts

```diff
@@ -20,4 +20,8 @@ Each component has general guidelines for usage as well as additional informatio
 - 🖥 denotes desktop specific information
   `,
 });
-extensionPoints(paths);
\ No newline at end of file
+extensionPoints(paths);
+
+adminExtensionApi(paths, {
+  componentsToSkip: ['ContainerApi', 'DataApi']
+});
```

Comment:
Missing an empty line at the end of the file

Reply:
we had eslint disabled for `scripts/`, i've enabled it which has fixed a whole bunch of formatting issues including this. couldn't enable it for `scripts/argo-admin`, but you're removing those right?